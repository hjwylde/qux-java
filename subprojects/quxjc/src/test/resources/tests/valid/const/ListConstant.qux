package valid.const

[int] A is [0, 1]
[int] B is [0, 1]
[int] C is [5]

[int] f([int] a):
    return a + A

[int] g([int] A):
    return A

void main([str] args):
    print A
    print B
    print C
    print A == B
    print A == C
    print B == C
    print f(C)
    print g(C)

