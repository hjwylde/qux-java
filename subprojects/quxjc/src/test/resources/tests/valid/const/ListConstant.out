[0, 1]
[0, 1]
[5]
true
false
false
[5, 0, 1]
[5, 5]
